_includes/sidebar.html

```diff
@@ -6,14 +6,15 @@
       <li><a href="/setup/">Install</a></li>
       <li><a href="/getting-started/">Create and run your first app</a></li>
       <li><a href="/bootstrap-into-dart/">Bootstrap into Dart</a></li>
-
+      <li><a href="https://codelabs.developers.google.com/codelabs/from-java-to-dart/index.html#0">Intro to Dart for Java Developers - Codelab</a></li>
```

Comment:
What do we think about adding this link to the bootstrap into dart page? I'd like to limit the number of "Get started" links, to minimize the choices our developers need to make in the first hour.